arches/app/templates/base-manager.htm

```diff
@@ -244,6 +245,25 @@ <h5>{% trans "for more documentation, visit" %} <a href="https://arches.readthed
                                         </li>
                                         {% endif %}
 
+                                        
+                                        <li>
+                                            <a href="{% url 'search_home' %}" class="ep-tools ep-tools-right" data-bind="click:function () { navigate('{% url 'search_home' %}') }">
```

Comment:
the classes added here (`class="ep-tools ep-tools-right"`) are causing some rendering issues - I think you should just remove them